src/main/java/frc/robot/commands/HopperForwardCommand.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.commands;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.subsystems.HopperSubsystem; 
+
+public class HopperForwardCommand extends CommandBase {
+  /** Creates a new IntakeForwardCommand. */
+  private final HopperSubsystem mHopperSubsystem;
+  
+  public HopperForwardCommand(HopperSubsystem hopperSubsystem) {
+    //Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(hopperSubsystem); 
+    mHopperSubsystem = hopperSubsystem;
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    mHopperSubsystem.HopperSwitch(true);
+    mHopperSubsystem.ForwardHopper();
```

Comment:
This could momentarily cause the motors to spin backwards and then change direction. Switch the order of these lines.